### PR TITLE
In process hardhat node

### PR DESCRIPTION
**Pull request by filippetroviccc**
Reference for #37

### CODE REVIEW

**Comment by fvictorio** [comment]
Is this ready for review? Because it only seems to modify the e2e tests :confused:

**Comment by filippetroviccc** [comment]
> Is this ready for review? Because it only seems to modify the e2e tests 😕

Yes, it should only modify e2e tests. Do you see any reason for this to be somewhere else also?

**Comment by fvictorio** [comment]
e2e tests don't seem to pass on my machine. Do I need to do something besides installing dependencies and building the rest of the projects?

And related to that: we should run them in the CI.

File: packages/e2e-tests/.mocharc.yaml [hunk 1]
```diff
@@ -6,4 +6,3 @@ require:
   - 'ts-node/register'
 recursive: true
 exit: true
-forbid-only: true
```
Comment (fvictorio) [inline packages/e2e-tests/.mocharc.yaml:9]: I frequently use `.only` while writing tests, that's why I removed this. But we can add it as a mocha param, for example having two scripts like this:

```
"test": "mocha ...",
"test:ci": "yarn test --forbid-only"
```

Alternatively, we can use a `.mocharc.js` and do something like:

```
forbidOnly: process.env.CI === 'true'
```

**Pull request merged**